Headers/DebugServer2/Core/HardwareBreakpointManager.h

```diff
@@ -67,7 +67,13 @@ class HardwareBreakpointManager : public BreakpointManager {
                                 std::vector<uint64_t> &regs) const;
 
 protected:
-  std::vector<uint64_t> _locations;
+  struct Location {
+    Location()
+      :address(0), value(0){}
```

Comment:
You'll need to run clang-format on this. You can either:
1) Install and run clang-format manually (I don't think this works on devservers)
2) Look at the Travis "STYLE" failures and manually apply
3) install clang-format and run: TARGET=Style ./Support/Testing/Travis/script.sh (see previous comment about no devservers, this will probably require temporarily changing script.sh to use an unversioned clang-format)